src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -500,9 +507,20 @@ public String getFullAppPath() {
         if (appFile == null) {
             return null;
         }
-
+        File dir = new File(appFile);
         Path path = Paths.get(appFile);
-        if (appFile.startsWith("sauce-storage:") || path.isAbsolute()) {
+        if (dir.isDirectory()) {
+            try {
+                String expectedFileExtension = platformName == Platform.ANDROID ? ".apk" : ".app";
+                File[] files = dir.listFiles((dir1, name) -> name.toLowerCase().endsWith(expectedFileExtension));
+                if (files != null) {
+                    File fileFoundMatchingExtension = files[0];
```

Comment:
Should you add any logic in the case that is more than one match? This could potentially return the full file path to an incorrect APK or APP file without such logic.

Reply:
I originally had logic to throw an error telling the user to specify an absolute path since there were more than 1 applications found, I forgot why I removed it -- let me think on this some more and try to implement something

Reply:
Addressed in https://github.com/willowtreeapps/conductor-mobile/pull/110/commits/74711e1d0d1e62e9bce6d4d5bd770dd9b21d06a6

Reply:
I logged it as an error but didn't destructively throw anything, if that is the right wording for it. Let me know if you think we should be more aggressive! 

Image of Error
![image](https://user-images.githubusercontent.com/22263679/62598293-efe79100-b8b6-11e9-8d27-78f51b605368.png)

Reply:
Perfect, this will prevent any possible confusion in the future. Thanks!